thrifty-kotlin-codegen/src/main/kotlin/com/microsoft/thrifty/kgen/KotlinCodeGenerator.kt

```diff
@@ -2219,6 +2225,17 @@ class KotlinCodeGenerator(
         return block.build()
     }
 
+    private fun makeFileSpecBuilder(packageName: String, fileName: String): FileSpec.Builder {
```

Comment:
I started with a `FileSpec.Builder.addJvmName` extension function, but once I made this optional behavior, I thought it made more sense to wrap the entire `FileSpec.Builder` creation pattern in a single function that either added the `JvmName` annotation or didn't.

Happy to use a different pattern based on feedback, though!

Reply:
This seems fine to me!